network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/primary_heroguts.html

```diff
@@ -3,9 +3,20 @@
 <div id="hero" class="mb-md-3">
   <div class="banner">
     {% if page.banner %}
-    {% image page.banner original alt="" %}
+    <picture>
+      {% image page.banner fill-4032x1152 as image_xl %}
+      {% image page.banner fill-2400x686 as image_lg %}
+      {% image page.banner fill-1984x567 as image_md %}
+      {% image page.banner fill-1536x439 as image_sm %}
+      <source media="(min-width: 1200px)" srcset="{{ image_xl.url }}" alt="Banner">
+      <source media="(min-width: 992px)" srcset="{{ image_lg.url }}" alt="Banner">
+      <source media="(min-width: 768px)" srcset="{{ image_md.url }}" alt="Banner">
+      <source media="(min-width: 576px)" srcset="{{ image_sm.url }}" alt="Banner">
+      {# Fallback Image #}
+      {% image page.banner fill-1008x288 alt="Banner" %}
+    </picture>
     {% else %}
-    <img src="/_images/banner-black-white-marble.jpg" alt="">
+    <img src="/_images/banner-black-white-marble.jpg" alt="Banner">
```

Comment:
We could. But in that case we'll have to rely on CMS editors to always provide meaningful alt text. Ummm... I can't decide which way we wanna go.

Reply:
Ah! I see that we are already doing this on the CMS for `ImageBlock`. Are these banners seen as decorative images? They are to me (as they are patterns) so I was debating if we should even provide alt text for screen reader to understand. Would love to hear your thoughts! @youriwims

Reply:
Hmmmm, I see the dilemma—I think in this case the banners are decorative. [WebAim](https://webaim.org/techniques/alttext/) says that if an image is solely for design purposes and doesn't convey any meaning then it should be removed from the content flow of the page and instead added via CSS. Can this image instead be added as a background image to remove the need for `alt` text?

Reply:
`aria-hidden="true"` is another option if this image works best in the content flow.

Reply:
@youriwims we can pass the image path to CSS without JS... unless we wanna use inline styling which is bad practice we should avoid anyway.

I'll add `aria-hidden="true"`. Do we still need alt text?

Reply:
Sorry, if you're unable to use CSS for the image, instead use the null `alt` attribute that you originally had since that'll be better supported in this case.